content/js/quick-glance.md

```diff
@@ -0,0 +1,40 @@
+The code below can give you a taste of what you can do with Skygear JS SDK.
+
+## Creating users
+
+``` javascript
+skygear.signupWithUsername(username, password).then(() => {
+  console.log(skygear.currentUser);
+}, (error) => {
+  console.error(error);
+});
+```
+
+Learn how to [Authentication Users](/js/guide/users). 
```

Comment:
It will be better to use "Learn more in Authentication". The link title should follow the menu